_maps/RandomRuins/LavaRuins/lavaland_surface_pride.dmm

```diff
@@ -73,9 +73,7 @@
 /area/ruin/powered/pride)
 "Y" = (
 /obj/machinery/door/airlock/diamond,
-/turf/open/floor/mineral/silver{
-	blocks_air = 1
-	},
```

Comment:
Bit late on this, but I feel like this should've gotten an invisible tiny fan as a replacement?